codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/protocol/aws/AwsJson10ProtocolGenerator.java

```diff
@@ -17,23 +17,24 @@
 
 import static software.amazon.smithy.go.codegen.ApplicationProtocol.createDefaultHttpApplicationProtocol;
 import static software.amazon.smithy.go.codegen.GoWriter.goTemplate;
+import static software.amazon.smithy.go.codegen.serde.SerdeUtil.getShapesToSerde;
 
 import java.util.Map;
+import java.util.Set;
 import software.amazon.smithy.aws.traits.protocols.AwsJson1_0Trait;
 import software.amazon.smithy.go.codegen.ApplicationProtocol;
 import software.amazon.smithy.go.codegen.GoWriter;
 import software.amazon.smithy.go.codegen.SmithyGoDependency;
-//import software.amazon.smithy.go.codegen.SmithyGoTypes;
```

Comment:
Don't commit commented out code. It's fine when testing locally, but it makes the code harder to follow. Remove the commented imports

Reply:
I can't find this import in my files? Maybe I made a change after committing. Sorry about that.

Reply:
This remains unaddressed.

Reply:
I seriously can't find this import - am I missing something here? I also checked it in vim and don't see it.

![Screenshot 2024-08-08 at 2 56 52 PM](https://github.com/user-attachments/assets/8468ff02-37b9-42d2-9cea-e6366193261c)

Reply:
Ignore me, I was tripped up by the diff. They're 100% not there anymore in the remote.